index.bs

```diff
@@ -7992,9 +7989,9 @@ a reference to the same object that the IDL value represents.
         starting at byte offset |offset|.
 </div>
 
-<div algorithm="detach an arraybuffer">
+<div algorithm>
 
-    To [=detach=] an {{ArrayBuffer}}, these steps must be followed:
+    To <dfn id="dfn-detach" for="ArrayBuffer" export>detach</dfn> an {{ArrayBuffer}}, these steps must be followed:
 
     1.  Let |O| be the ECMAScript object that is the {{ArrayBuffer}}.
     1.  [=DetachArrayBuffer=](|O|).
```

Comment:
While here, maybe change this to `Perform [=!=] [=DetachArrayBuffer=](|O|)` as part of the modernization process